content/values/open-source.md

```diff
@@ -0,0 +1,14 @@
++++
+# Homepage
+type = "blank"
+headless = true  # Homepage is headless, other widget pages are not.
+weight = 10
+title = "Open source tools for research and education"
+
+[design]
+  columns = "1"
++++
+
+2i2c supports and uses projects that are developed, controlled, and owned by the community. We believe in the importance of tools that are built by a diverse community of multiple stakeholders, in order to build tools that are powerful for all of us, and in an equitable and inclusive manner.
+
+Open source tools exist across all parts of the data science stack - from the domain-specific libraries you use to analyze data, to the interfaces that allow you to interact with the computer, to the software that orchestrates your workflow in the cloud. 2i2c believes it is crucial that research and education has access to open source tools across **all layers of this stack**.
```

Comment:
I love this! <3

The only thing that I'd rephrase is the `has access` in this sentence:
> 2i2c believes it is crucial that research and education has access to open source tools across **all layers of this stack**.

In my mind, everyone has access to all the open source tools, by definition <3, and what 2i2c does, is rather to put these together and give access to the end machinery.

(Hope I made some sense and this didn't come out as an undecipherable idea :sweat_smile:)